pages.id/common/docker-ps.md

```diff
@@ -0,0 +1,36 @@
+# docker ps
+
+> Menampilkan daftar kontainer Docker.
+> Informasi lebih lanjut: <https://docs.docker.com/engine/reference/commandline/ps/>.
+
+- Menampilkan kontainer Docker yang sedang berjalan saat ini:
+
+`docker ps`
+
+- Menampilkan semua kontainer Docker (yang berjalan dan yang berhenti):
+
+`docker ps --all`
+
+- Menampilkan kontainer yang dibuat terakhir (termasuk semua status):
+
+`docker ps --latest`
+
+- Memilah kontainer yang mengandung substring dalam namanya:
+
+`docker ps --filter="name={{nama}}"`
+
+- Memilah kontainer yang memiliki gambar yang sama sebagai leluhur:
+
+`docker ps --filter "ancestor={{image}}:{{tag}}"`
+
+- Memilah kontainer berdasarkan kode status keluar (exit status code):
+
+`docker ps --all --filter="exited={{kode}}"`
+
+- Memilah kontainer berdasarkan status (created, running, removing, paused, exited, dan dead):
+
+`docker ps --filter="status={{status}}"`
+
+- Memilah kontainer yang mengaitkan suatu volume tertentu atau memiliki volume yang terpasang pada jalur tertentu:
+
+`docker ps --filter="volume={{path/ke/direktori}}" --format "table {{.ID}}\t{{.Image}}\t{{.Names}}\t{{.Mounts}}"`
```

Comment:
Thanks for your contribution, I have a suggestion for this page.

```suggestion
`docker ps --filter="volume={{jalan/menuju/direktori}}" --format "table {{.ID}}\t{{.Image}}\t{{.Names}}\t{{.Mounts}}"`
```
Ref. [Common Arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)